extensions/binding/org.eclipse.smarthome.binding.yahooweather/src/main/java/org/eclipse/smarthome/binding/yahooweather/handler/YahooWeatherHandler.java

```diff
@@ -55,6 +51,12 @@
 
     private final int MAX_DATA_AGE = 3 * 60 * 60 * 1000; // 3h
     private final int CACHE_EXPIRY = 10 * 1000; // 10s
+    private final ExpiringCacheMap<String, String> CACHE = new ExpiringCacheMap<String, String>(CACHE_EXPIRY);
```

Comment:
@martinvw Good point. I think we failed to see that. I will fix it in a follow-up PR.